apollo-backend/src/test/java/io/logz/apollo/SingleRegionBlockerTest.java

```diff
@@ -123,23 +123,27 @@ public void testSingleRegionBlockerAllowsDeploymentsWithDifferentAvailability()
         Environment env2 = ModelsGenerator.createEnvironment(availabilityStaging, null);
         env2.setId(apolloTestClient.addEnvironment(env2).getId());
 
+        Environment env3 = ModelsGenerator.createEnvironment(availabilityStaging, null);
+        env3.setId(apolloTestClient.addEnvironment(env3).getId());
+
         ModelsGenerator.createAndSubmitPermissions(apolloTestClient, Optional.of(env1), Optional.empty(), DeploymentPermission.PermissionType.ALLOW);
         ModelsGenerator.createAndSubmitPermissions(apolloTestClient, Optional.of(env2), Optional.empty(), DeploymentPermission.PermissionType.ALLOW);
+        ModelsGenerator.createAndSubmitPermissions(apolloTestClient, Optional.of(env3), Optional.empty(), DeploymentPermission.PermissionType.ALLOW);
 
         blocker = createAndSubmitBlocker(apolloTestAdminClient, BlockerTypeName.SINGLE_REGION, "{}", null, serviceToBeLimitToOneRegion, null, env1.getAvailability());
 
-        List<Environment> prodAndStagingEnvironments = Arrays.asList(env1, env2);
-
         List<Callable<MultiDeploymentResponseObject>> deploymentRequests = new ArrayList<>();
-        addDeploymentRequests(apolloTestClient, serviceToBeLimitToOneRegion, deployableVersion, prodAndStagingEnvironments, deploymentRequests);
+
+        deploymentRequests.add(() -> apolloTestClient.addDeployment(String.valueOf(env1.getId()), String.valueOf(serviceToBeLimitToOneRegion.getId()), deployableVersion.getId()));
+        deploymentRequests.add(() -> apolloTestClient.addDeployment(env2.getId() + "," + env3.getId(), String.valueOf(serviceToBeLimitToOneRegion.getId()), deployableVersion.getId()));
```

Comment:
please add comment above: `// deploying multiple environments in staging shouldn't be affected by SRB in prod`

Reply:
done